### PR TITLE
Story 53

### PR DESCRIPTION
**Pull request by agallant121**
Complete user story 53

### CODE REVIEW

File: app/controllers/admin/admin_users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class Admin::AdminUsersController < Admin::BaseController
+  def index
```
Comment (agallant121) [inline app/controllers/admin/admin_users_controller.rb:2]: Add AdminUsers controller for admins to see all users

File: app/views/admin/admin_users/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<h1>Users Index:</h1>
+
+<% @users.each do |user| %>
```
Comment (agallant121) [inline app/views/admin/admin_users/index.html.erb:3]: Add users index spec

File: spec/features/admins/admin_users_index_spec.rb [hunk 3]
```diff
@@ -0,0 +1,104 @@
+require 'rails_helper'
+
```
Comment (agallant121) [inline spec/features/admins/admin_users_index_spec.rb:2]: Add spec to check if admin can see a users index and that only the admins can access the users index

**Comment by palworth** [comment]
Ready for Master.

**Pull request merged**